### DISCUSSION TITLE
Support timestamp manipulation in template variables

### DISCUSSION THREAD

**Discussion by wardbekker** [grafana/grafana#44026] [Dashboards, Panels & Visualizations]
Request to support timestamp manipulation in template variables. This will allow time addition and subtraction in datalinks like this: 

https://foo.grafana.net/dashboard/new?orgId=1&from=${__from-1minute}&to=${__from+1minute}

This makes it possible to have datalinks with timewindows that are a bit larger or smaller than the original from-to of the dashboards.

**Comment by nicolaerosia** [comment]
this would be nice!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
